mq/plugins/squidFixup.py

```diff
@@ -71,16 +73,12 @@ def onMessage(self, message, metadata):
         if message["category"] != "proxy":
             return message, metadata
 
-        # Reuse the NSM doc type
```

Comment:
We should probably copy over these comments, since it seems weird that we're setting the type to nsm for squid events.

Reply:
Oh that must be leftover from when michal copied a brofixup to make this one. We should modify that type to be squid.

Reply:
I've corrected this by setting it to squid